.eslintignore

```diff
@@ -0,0 +1 @@
+app/webpacker/src/javascripts/commodities.js
```

Comment:
:nauseated_face:

Reply:
Yeh, I feel like that - bit wary of creating a lot of noise and potential regressions in a fragile poorly tested file

Linting alone will not make that file any less awful, and if it gets refactored then anything new replacing it will still need to comply with linting rules